Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -2619,6 +2619,20 @@ FileSystemAccessEnabled:
       "PLATFORM(COCOA)" : true
       default: false
 
+FilterLinkDecorationEnabled:
```

Comment:
Since LDF is already enabled in Private Browsing by default, perhaps we could call this something like "FilterLinkDecorationInNormalBrowsingEnabled" or "AlwaysFilterLinkDecorationEnabled"?